feature-libs/product/future-stock/components/future-stock-accordion/future-stock-accordion.component.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Input } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { FutureStock } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-future-stock-accordion',
+  templateUrl: './future-stock-accordion.component.html',
+})
+export class FutureStockAccordionComponent {
+  @Input() header: string;
+  @Input() content: FutureStock[] | string;
+
+  expanded: boolean = false;
```

Comment:
Lets handle this with a behaviour subject so that the template can respond more efficiently to changes.